pydream/Dream.py

```diff
@@ -70,7 +70,13 @@ def __init__(self, model, variables=None, nseedchains=None, nCR=3, adapt_crossov
         self.mp_context = mp_context
         # Set model and variable attributes (if no variables passed, set to all parameters)
         self.model = model
-        self.model_name = model_name
+        if not model_name:
+            # Replace `:` for `_` because colons cause problems in windows paths
+            prefix = datetime.now().strftime('%Y_%m_%d_%H:%M:%S').replace(':', '_') + '_'
```

Comment:
No need to `replace` separately; just use underscores in the `strftime` call. You can attach the tailing underscore in that call, too.

Reply:
I thought that when I tried it, strftime required the time to be with the colons. I just double-checked that and you are correct. I will fix that accordingly. Thanks!